Add descriptive suffix to frameworks.

shoumikhin opened the pull request:
Summary: .

Differential Revision: D56868042

mergennachin commented on .gitmodules:
```diff
@@ -68,3 +68,6 @@
 [submodule "examples/models/llama2/third-party/abseil-cpp"]
 	path = examples/models/llama2/third-party/abseil-cpp
 	url = https://github.com/abseil/abseil-cpp.git
+[submodule "third-party/ios-cmake"]
+	path = third-party/ios-cmake
+	url = https://github.com/leetal/ios-cmake
```
is this necessary? 

in the phabricator version i didn't see these changes

shoumikhin replied:
I guess when exported the stacked diffs get flattened in GitHub PRs.

shoumikhin committed:
Build Mac frameworks. (#3429)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/3429

Differential Revision: D56778448

Reviewed By: cccclai

Pulled By: shoumikhin

shoumikhin committed:
Add descriptive suffix to frameworks. (#3479)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/3479

.

Reviewed By: mergennachin

Differential Revision: D56868042
